docs/connect-cloudant-source-example.properties

```diff
@@ -0,0 +1,15 @@
+# sample configuration, source connector
```

Comment:
In conjunction with above suggestion to make the source type more clear i.e. renaming to `docs/connect-cloudant-changes-source-example.properties` and update comment
```suggestion
# sample configuration, changes source connector
```